docs/src/installation_instructions.md

```diff
@@ -1,25 +1,81 @@
 release. Oceananigans.jl can be updated to the latest tagged release from the package manager by typing
 
 ```julia
-(v1.6) pkg> update Oceananigans
+(new_project) pkg> update Oceananigans
 ```
 
-At this time, updating should be done with care, as Oceananigans is under rapid development and breaking changes to the user API occur often. But if anything does happen, please open an issue!
-
-But if anything does happen or your code stops working, please open an issue and ask! We're more than happy to help with getting your simulations up and running.
+At this time, updating should be done with care, as Oceananigans is under rapid development and breaking 
+changes to the user API occur often. But if anything does happen, please open [an issue on github](https://github.com/CliMA/Oceananigans.jl/issues)!
+We're more than happy to help with getting your simulations up and running.
 
 !!! compat "Julia 1.6 or newer"
     The latest version of Oceananigans requires at least Julia v1.6 to run.
     Installing Oceananigans with an older version of Julia will install an older version of Oceananigans (the latest version compatible with your version of Julia).
+
+## Running Oceananigans
+
+Whenever you run Oceananigans for a project, it's recommended that you start Julia not with the simple command `julia`,
+but with the command `julia --project`. This ensures that you will always use the package versions of you project
+and that any changes there will not affect other projects. That said, running Oceananigans can be done a couple of ways. 
+
+
+For initial explorations and getting used to the code, you can
+enter the commands directly in the [Julia REPL](https://docs.julialang.org/en/v1/stdlib/REPL/) (which can be started with
+the command `julia --project`). This is an interactive
+command-line tool (similar to Python's IPython) that has a searchable history, tab-completion, helpful keybindings, and many
+other features to help users interact with packages. If you already have a file with commands, you can issue `julia --project my_oceananigans_script.jl`
+from the terminal, which will run all the commands in that script and subsequently close Julia. This is equivalent to opening
+the Julia REPL inside your project directory and typing each of those commands by hand (or simply issuing
+`include("my_oceananigans_script.jl` in the REPL), with the exception that the REPL will remain open after the commands in this case.
+
+If you are using a HPC system and you want to make use of a GPU, then an extra step generally has to be added: submitting the
+job to request GPU resources. This is usually done with the `slurm` or `PBS` job schedulers and the details usually change depending
+on which cluster you are using, but the process should be similar (if not the same) to submitting other jobs in that same cluster. We
+recommend you check the cluster documentation for information on how to submit jobs that require GPU resources.
```

Comment:
This has nothing to do with Oceananigans.

Reply:
True, which is why there's a referral to whatever HPC system the user is using. But I'd argue that's useful. Having that part there would definitely have made my life easier 2 years ago when I started using Oceananigans.

Although I agree that we might wanna tie that with the act of "running Oceananigans" a bit more

Reply:
@navidcy check this newer version which is more brief. I think that's useful for new users, but if you still feel like that's out of place, then I'm okay removing it.

Reply:
I tend to agree that we shouldn't put info here that's not relevant to Oceananigans. There's an infinite number of things that might make all our lives easier...

Reply:
Perhaps more to the point, if there is a need for "tips for using GPUs with slurm and PBS (and Julia?)" then that information belongs elsewhere. It's _less_ helpful to all those suffering humans to bury this info in the Oceananigans docs!

Reply:
Yes. Let's drop this bit.